client/blocks/reader-post-card/style.scss

```diff
@@ -212,6 +240,11 @@ $reader-post-card-breakpoint-small: "( max-width: 550px )";
 	}
 }
 
+.reader-post-card__meta-separator {
```

Comment:
@bluefuton Do you think we need a new span for this? Perhaps just an `&::after` for the Author (with a comma as a content) would work?